packages/reports/src/Containers/open-positions.jsx

```diff
@@ -360,6 +361,10 @@ const OpenPositions = ({
         contract_types.find(type => type.is_default)?.value || 'Options'
     );
 
+    const accumulator_rates = ['all', '0.01', '0.02', '0.03', '0.04', '0.05'];
+
+    const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
```

Comment:
```suggestion
    const accumulator_rates = ['all', '0.01', '0.02', '0.03', '0.04', '0.05'];
    const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
```
@henry-deriv nit-pick: just some extra line breaks😊